mozdownload/__init__.py

```diff
@@ -2,26 +2,31 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+__version__ = '1.19'
+
```

Comment:
We never do version bumps while other stuff is still open to be fixed for the next milestone. So the bump will happen with the release issue/PR.